portlets/src/main/resources/locale/portlet/Analytics_en.properties

```diff
@@ -17,6 +17,7 @@ analytics.field.label.announcementActivityId=Announcement Activity Id
 analytics.field.label.announcementAssignee=Announcement assignee
 analytics.field.label.announcementComment=Announcement comment
 analytics.field.label.announcementComment=Announcement comment
+analytics.field.label.objectType= Gamification: Object type
```

Comment:
collision with the label defined in Tasks, the same key with two different labels.

Reply:
done!